contents.json

```diff
@@ -2522,6 +2522,18 @@
       ]
     },
     {
+      "title": "Link Keyboard : My Links Everywhere",
+      "category-ids": [
+        "keyboard",
+        "extension"
```

Comment:
can you remove extension, even though it is correct, these types of apps are only keyboard right now